app/uk/gov/hmrc/gform/des/DesConnector.scala

```diff
@@ -31,17 +37,56 @@ import uk.gov.hmrc.http.logging.Authorization
 class DesConnector(wSHttp: WSHttp, baseUrl: String, desConfig: DesConnectorConfig) {
 
   def lookupRegistration(utr: String, desRegistrationRequest: DesRegistrationRequest)(
-    implicit ex: ExecutionContext): Future[DesRegistrationResponse] = {
+    implicit ex: ExecutionContext): Future[ServiceCallResponse[DesRegistrationResponse]] = {
```

Comment:
you can just call the class "ServiceResponse"

Reply:
`ServiceResponse` is data constructor for `ServiceCallResponse` data type. (here I'm using haskell vocabulary)